backend/schema.js

```diff
@@ -74,6 +74,12 @@ type Vote {
     entry: Entry
```

Comment:
these properties are not resolving currently

To add:

1. add a `backend/resolvers/types/voteType.js` which defines `judge` and `entry` properties somehow, see [`backend/resolvers/types/groupType.js`](https://github.com/robmcl4/A-B-S-T-R-A-C-T-I-O-N/blob/master/backend/resolvers/types/groupType.js) as an example. You may also find it useful to define some helper methods on `Vote`.
2. add the above file to `backend/resolvers/index.js`

we haven't exactly figured out to test this behavior yet, so don't worry it's easy to overlook :/